Bottom: 0b29982bdc35cf0bf6f6323b4db0e5e63b5b03c7
Top:    788248f96ef837f6d6b255e315cdd92de8859413
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 16:38:41 +0000

Refresh of add-strip-hierarchy-options-to

---

diff --git a/src/action.c b/src/action.c
index b85c88b..bb75808 100644
--- a/src/action.c
+++ b/src/action.c
@@ -4798,10 +4798,10 @@ ActionToggleStripHierarchy (int argc, char **argv, Coord x, Coord y)
 				      &ptr1, &ptr2, &ptr3)) != NO_TYPE)
 	      {
 		AddObjectToFlagUndoList (type, ptr1, ptr2, ptr3);
-		EraseElementName ((ElementTypePtr) ptr2);
-		TOGGLE_FLAG (HIDENAMEFLAG, (ElementTypePtr) ptr2);
+		EraseElementName ((ElementType *) ptr2);
+		TOGGLE_FLAG (HIDENAMEFLAG, (ElementType *) ptr2);
 /* TODO: 		   SetTextBoundingBox (&PCB->Font, new); */
-		DrawElementName ((ElementTypePtr) ptr2);
+		DrawElementName ((ElementType *) ptr2);
 		Draw ();
 		IncrementUndoSerialNumber ();
 	      }
diff --git a/src/draw.c b/src/draw.c
index c7e849d..5b9bd0a 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -152,7 +152,7 @@ pad_callback (const BoxType * b, void *cl)
 }
 
 static void
-DrawStrippedText (ElementTypePtr Element, int min_width)
+DrawStrippedText (ElementType *Element, int min_width)
 {
   TextType text;
   TextType *text_ptr;
